packages/router/__tests__/router-test.ts

```diff
@@ -3320,7 +3320,7 @@ describe("a router", () => {
       expect(t.router.state.navigation.formData).toBeUndefined();
     });
 
-    it("merges URLSearchParams for formMethod=get", async () => {
+    it("does not preserve existing 'action' URLSearchParams for formMethod='get'", async () => {
```

Comment:
This was a previous misunderstanding on my part - browser do indeed blow away query params on GET submissions.  I thought it would just append them